### PR TITLE
feat: render default slot in stubs

### PR DESCRIPTION
**Pull request by eddyerburgh**
Render slots in default stubs.

Breaking change

### CODE REVIEW

**Review by lmiller1990 — COMMENTED** [review]
Looks good to me - question, what impact will this have on the public api?

What is the procedure for a breaking change like this? Adding a notice to people using the version prior to the breaking changes?

**Comment by eddyerburgh** [comment]
It's a breaking change because anyone who relies on shallowMount not including slot children will have to change their code.

Because VTU is still in beta, we can release a breaking change in the next version. Once VTU is 1.x we will only add breaking changes in major versions.

File: packages/shared/stub-components.js [hunk 1]
```diff
@@ -79,7 +79,9 @@ function createBlankStub (originalComponent: Component) {
   return {
     ...getCoreProperties(originalComponent),
     render (h) {
-      return h(name)
+      return h(`${originalComponent.name}-stub`,
```
Comment (38elements) [inline packages/shared/stub-components.js:82]: I think this line does not need to change.

```js
const name = `${originalComponent.name}-stub`
```

**Comment by eddyerburgh** [comment]
Closed in favor of https://github.com/vuejs/vue-test-utils/pull/782

**Pull request closed without merging**